ansible/roles/dokuwiki/files/libexec/update_user

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import os
+from argparse import ArgumentParser
+import codecs
+import hashlib
+from collections import OrderedDict
+from pathlib import Path
+
+
+def ssha(pw):
+    salt = os.urandom(4)
```

Comment:
I agree, but I'd also like to merge this asap. If you feel strongly about this, let's create an issue, how's that?